Plugins/src/main/java/tw/waterball/judgegirl/plugins/impl/cqi/CodeQualityInspectionPlugin.java

```diff
@@ -103,10 +100,10 @@ private CyclomaticComplexityReport calcCyclomaticComplexity(Path sourceRootPath)
         return null;
     }
 
-    @SneakyThrows
-    private String readSourceCode(String sourceCodePath) {
-        return Files.readString(Paths.get(sourceCodePath));
-    }
+//    @SneakyThrows
+//    private String readSourceCode(String sourceCodePath) {
+//        return Files.readString(Paths.get(sourceCodePath));
+//    }

```

Comment:
why?

Reply:
Keep it commented, that's not used for a while